stdeb.cfg

```diff
@@ -15,8 +15,8 @@ X-Python3-Version: >= 3.4
 Setup-Env-Vars: SKIP_PYTHON_MODULES=1
 
 [rospkg_modules]
-Depends: python-catkin-pkg-modules, python-yaml, lsb-release
-Depends3: python3-catkin-pkg-modules, python3-yaml, lsb-release
+Depends: python-catkin-pkg-modules, python-distro, python-yaml, lsb-release
+Depends3: python3-catkin-pkg-modules, python3-distro, python3-yaml, lsb-release
```

Comment:
I could only find the Debian packages for Ubuntu Bionic and newer:
* https://packages.ubuntu.com/search?keywords=python-distro
* https://packages.ubuntu.com/search?keywords=python3-distro

This package needs to target all non-EOL platforms that atm go back until Ubuntu Xenial / Debian Jessie.